bin/run

```diff
@@ -35,29 +65,38 @@ init_connections() {
         --conn_host https://dbc-caf9527b-e073.cloud.databricks.com \
         --conn_extra "{\"token\":\"${DB_TOKEN}\"}"
 
-    airflow connections --add \
-        --conn_id google_cloud_prio_admin \
-        --conn_type google_cloud_platform
-
-    airflow connections --add \
-        --conn_id google_cloud_prio_a \
-        --conn_type google_cloud_platform
-
-    airflow connections --add \
-        --conn_id google_cloud_prio_b \
-        --conn_type google_cloud_platform
-
-    airflow connections --add \
-        --conn_id amplitude_s3_conn \
-        --conn_type s3
-
-    airflow connections --add \
-        --conn_id google_cloud_derived_datasets \
-        --conn_type google_cloud_platform
-
-    airflow connections --add \
-        --conn_id google_cloud_airflow_dataproc \
-        --conn_type google_cloud_platform
+    gcp_conn=(
+      "google_cloud_airflow_dataproc"
+      "google_cloud_derived_datasets"
+      "google_cloud_prio_a"
+      "google_cloud_prio_admin"
+      "google_cloud_prio_b"
+    )
+    for conn_id in "${gcp_conn[@]}"; do
+        airflow connections --delete --conn_id "${conn_id}"
+        airflow connections --add \
+        --conn_id "${conn_id}" \
+        --conn_type google_cloud_platform \
+        --conn_extra "$(gcp_default_extras)"
```

Comment:
@fbertsch This is how the GCP specific parameters can be passed into the connection string via the CLI. I'm not entirely sure that this solves your problem, but there's now a way to provide a dummy keyfile_dict and project_id.